src/software/ai/hl/stp/play/stop_play_test.cpp

```diff
@@ -23,13 +23,12 @@ class StopPlayTest : public SimulatedPlayTestFixture
         return {
             [](std::shared_ptr<World> world_ptr, ValidationCoroutine::push_type& yield) {
                 // Wait 2 seconds for robots that start too close to the ball to move away
-                while (world_ptr->getMostRecentTimestamp() < Timestamp::fromSeconds(2))
+                if (world_ptr->getMostRecentTimestamp() > Timestamp::fromSeconds(2))
```

Comment:
Minor nit, but the penalty kick play test checks if the timestamp is >= 2 seconds, can we either change this one to match or change the penalty kick to > 2 seconds, for consistency's sake?

Reply:
fixed!